core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnMLOp.scala

```diff
@@ -0,0 +1,73 @@
+package edu.uci.ics.texera.workflow.operators.sklearn
+
+import com.fasterxml.jackson.annotation.{JsonIgnore, JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+abstract class SklearnMLOp extends PythonOperatorDescriptor {
+  @JsonIgnore
+  var model = "from sklearn.dummy import DummyClassifier"
+
+  @JsonIgnore
+  var name = "Dummy Classifier"
+
+  @JsonProperty(value = "Target Attribute", required = true)
+  @JsonPropertyDescription("column in your dataset corresponding to target")
+  @AutofillAttributeName
+  var target: String = _
+
+  override def generatePythonCode(): String =
+    s"""$model
+       |from pytexera import *
```

Comment:
to avoid conflicts, please move `from pytexera import *` to the last import.

Reply:
Fixed.